cypress/integration/basic.spec.js

```diff
@@ -42,10 +42,38 @@ describe('default behavior', () => {
       .get('#ocd .trap')
       .should('have.class', 'is-active')
       .get('body')
-      .click()
+      .click({ force: true })
```

Comment:
What is this for? Can you add a comment above the line pointing out the reason we need a force?

Reply:
`click` will attempt to click on the middle of an element, so when we click on `body`, the vertical center is no longer visible on the page due to the increased length from me adding another example. I'll add a comment to provide the context.